### ISSUE TITLE
Site build fails 

### ISSUE THREAD

**Issue by milancurcic** [fortran-lang/fortran-lang.org#337]
Build for PR #334 fails: https://github.com/fortran-lang/fortran-lang.org/runs/3702605456. I think the offending line is https://github.com/fortran-lang/fortran-lang.org/runs/3702605456#step:7:103, which perhaps means that there is a required field in the package index that's missing?

What's more confusing to me, we only have some (inactive?) Twitter action running on open PRs. I thought we used to build the site in CI on every PR.

**Comment by certik** [comment]
At least the issue #298 now seems fixed. ;)

**Comment by milancurcic** [comment]
From the Action log, the Ruby parser complains about the gFTL entry : https://github.com/fortran-lang/fortran-lang.org/blob/a8dfa9716cb1fe5b50283ed07d5bd4f92612eec2/_data/package_index.yml#L1386.

The error message is

```
/home/runner/work/fortran-lang.org/fortran-lang.org/src/.bundle/ruby/2.7.0/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `parse': (/home/runner/work/fortran-lang.org/fortran-lang.org/src/_data/package_index.yml): did not find expected key while parsing a block mapping at line 1386 column 3 (Psych::SyntaxError)
```

But from looking at that block, it seems to me all needed keys are there. @awvwgk @LKedward do you have an idea?

**Comment by milancurcic** [comment]
Fixed by #336.

**Comment by certik** [comment]
I guess there was an extra space at the beginning of the line, so the yaml syntax was broken?
